docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,93 @@
+- Types moving from `Windows` to `Microsoft`:
+    - `Windows.UI.Xaml`
+    - `Windows.UI.Colors`
+    - `Windows.UI.ColorHelper` 
+    - `Windows.UI.Composition`
+    - `Windows.UI.Text`
+    - `Windows.System.DispatcherQueue` 
+    - `Windows.System.DispatcherQueueController`
+    - `Windows.System.DispatcherQueueHandler`
+    - `Windows.System.DispatcherQueuePriority`
+    - `Windows.System.DispatcherQueueShutdownStartingEventArgs`
+    - `Windows.System.DispatcherQueueTimer`
+- Converts `App.OnLaunched` Method
+    - Two updates need to be made to the `App.OnLaunched` method when converting to WinUI3:
+    1. Target `Microsoft.UI.Xaml.LaunchactivatedEvenArgs` as the method parameter type
+    2. Instances of the parameter name in the `App.OnLaunched` method body must invoke `UWPLaunchActivatedEventArgs`
+- Highlights Deprecated Types
+    - Some types such as `Windows.UI.Input.Inking`, and `Windows.UI.Xaml.Media.AcrylicBackgroundSource` are not supported in WinUI3. These may be identified by the analyzer as deprecated but cannot be converted automatically.
+
+### What The Conversion Analyzers Do Not Do:
+
+ - Modify .csproj files or modify/resolve conflicting NuGet packages
+ - Remove deprecated Code
+ - Analyze/Convert .xaml files
+
+## Examples
+### Convert Namespaces:
+Before converting:
+
+![Visual Studio Lightbulb Suggestion](images\namespaceBefore.png#thumb)
+
+After:
+
+![Visual Studio Lightbulb Suggestion](images\namespaceAfter.png#thumb)
+
+### Convert App.OnLaunched Method:
+Before converting:
+
+![Visual Studio Lightbulb Suggestion](images\onlaunchedBefore.png#thumb)
+
+After:
+
+![Visual Studio Lightbulb Suggestion](images\onlaunchedAfter.png#thumb)
+
+
+## Conversion Process
+Necessary steps for converting a WinUI C# app to WinUI3: 
+
+1. Install Microsoft.WinUI and WinUIConvert NuGet packages in your app using the NuGet package manager: see the [Install WinUI 3 Preview](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/#install-winui-3-preview-2) page for more information.
+    
+    ![Visual Studio Lightbulb Suggestion](\images\newPackage.png#thumb)
+
+2. Uninstall Microsoft.UI.Xaml from your solution. Additional conflicting packages such as `Microsoft.Xaml.Behaviors.*` may also need to be removed. 
+    
+    ![Visual Studio Lightbulb Suggestion](\images\uninstallOldPackage.png#thumb)
+
+3. Light Bulb suggestions should highlight issues that need to be updated for WinUI3 conversion. 
+
+    ![Visual Studio Lightbulb Suggestion](\images\lightbulb.png#thumb)
+
+4. Click the down arrow by the lightbulb, Convert to WinUI3 and select Fix all occurences in project. 
+
+    ![Visual Studio Lightbulb Suggestion](\images\analyzer-1.png#thumb)
+
+5. All namespace changes should be fixed in your project!
+
+6. The Analyzer can only parse C# code and not Xaml. WebView is now WebView2 and should be updated: see the [Getting Started](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/) page for more information.
+---
```

Comment:
After the port is done, should you have to remove the converter nuget package ?

Reply:
Added optional step reminding about NuGet packager uninstall